Documentation/QuickInstall/TheInstallTool/Index.rst

```diff
@@ -76,6 +76,13 @@ The Admin Tool
       Proceed with :ref:`install-introduction-package`
 
 
+#. In the last step you verify the image handling settings under the configuration presets 
+   for Image Magick and Graphics Magick.
+   E.g in Windows you must enter without any quotes ::
+
+     C:\Program Files\ImageMagick-7.0.11-Q8
```

Comment:
This part does not really make sense. What is the user supposed to then? If this should be a hint that IM/GM should be installed and this path should be set in TYPO3 then we should say so. However I assume this is already spelled out clearly by TYPO3 itself ...

Reply:
You should at least mention (using `:guilabel:` and/or `$GLOBALS['TYPO3_CONF_VARS']` path) where this path should be entered.